src/v/raft/tests/raft_group_fixture.h

```diff
@@ -467,7 +467,7 @@ struct raft_group {
         for (auto& [_, m] : _members) {
             close_futures.push_back(m.stop_node());
         }
-        ss::when_all(close_futures.begin(), close_futures.end()).get0();
+        ss::when_all_succeed(close_futures.begin(), close_futures.end()).get0();
```

Comment:
Can this now throw? Should we use try/catch or `handle_exception([](std::exception_tr e){ log(e); })`?

Reply:
With the fix in the other commit, this shouldn't throw any more -- if it does we want that to surface as a test failure.  I don't love throwing exceptions from destructors but I think the way these fixtures work the destructor is where we have to do the teardown.

Reply:
If the throw from destructor is intentional, then this is fine.